packages/sdk/rollup.config.js

```diff
@@ -0,0 +1,101 @@
+import json from '@rollup/plugin-json';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import typescript from 'rollup-plugin-typescript2';
```

Comment:
Let's use the official TypeScript plugin: https://www.npmjs.com/package/@rollup/plugin-typescript

Reply:
The official Typescript plugin has some weird quirks (unfortunately I can't remember the exact details), which is why I went with `rollup-plugin-typescript2` back in the days. If we are going to touch this, I think we should go directly with something like `rollup-plugin-esbuild`.